scripts/coccinelle.sh

```diff
@@ -7,7 +7,7 @@ FAILED=false
 true > coccinelle.diff
 
 for f in "${SCRIPT_DIR}"/../coccinelle/*.cocci; do
-	find "${SCRIPT_DIR}"/.. -name '*.c' -exec spatch --very-quiet -sp_file "$f" {} + | tee -a coccinelle.diff
+	find "${SCRIPT_DIR}"/.. \( -name '*.h' -or -name '*.c' \) -exec spatch --very-quiet -sp_file "$f" {} + | tee -a coccinelle.diff
```

Comment:
Nit
```suggestion
	spatch --very-quiet --sp_file "$f" --dir "${SCRIPT_DIR}"/.. | tee -a coccinelle.diff
```

Reply:
@mkindahl Does `--dir` work reliably on your system? On my system, it only checks `.c` files and ignores the header files.

Reply:
note: if you go back to using `find`:  `find . -name '*.[hc]'`

Reply:
@kgyrtkirk Thanks, changed.

Reply:
> @mkindahl Does `--dir` work reliably on your system? On my system, it only checks `.c` files and ignores the header files.

Sorry, you need to add the `--include-headers` option for this.

Reply:
@mkindahl I missed that option in the documentation. Using `--include-headers` it also works reliably with my version of `spatch`.

Reply:
Good. I already approved the patch so whichever approach you prefer is fine.

Reply:
I will use the version you suggested. This is much faster than starting `spatch` for each file individually.